core/unix/os_exports.h

```diff
@@ -246,8 +246,10 @@ ushort
 os_get_app_tls_reg_offset(reg_id_t seg);
 void *
 os_get_app_tls_base(dcontext_t *dcontext, reg_id_t seg);
+#if defined(AARCH64) || (defined(X64) && defined(X86))
 bool
-os_dump_core_live(void);
+os_dump_core_live(dcontext_t *dcontext, priv_mcontext_t *mc);
```

Comment:
Explain why this takes in a separate mc pointer: for internal use this can just be a pointer to the values inside the dcontext, right?  Make that clear in a comment.

Reply:
I've changed the code to pass in dcontext_t only.